ldk/node/.gitignore

```diff
@@ -1,51 +1,166 @@
-# npm & yarn
-node_modules/
-.npm
-.yarn-integrity
+#----------------Node--------------
+# Logs
+logs
+*.log
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-yarn.lock*
-/build/bundle.js
-/build/index.html
-/build*
-/build/*
-
-# Testing
-coverage/
+lerna-debug.log*
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
+# Runtime data
+pids
+*.pid
```

Comment:
Should we be making all these changes here instead of the root `.gitignore`? I think there might be some duplication between some of these entires

Reply:
Well. This is a standard .getignore, specific to node js. There should be no issues if it is duplicated.